packages/jest-haste-map/src/lib/extract_requires.js

```diff
@@ -12,8 +12,8 @@ const blockCommentRe = /\/\*[^]*?\*\//g;
 const lineCommentRe = /\/\/.*/g;
 
 const replacePatterns = {
-  EXPORT_RE: /(\bexport\s+(?:[^'"]+\s+from\s+)??)(['"])([^'"]+)(\2)/g,
```

Comment:
Nope, probably no tests :) Feel free to send one, it's the same as import, just that it exports also:

```
export foo from 'foo-module';
export type foo from 'foo-module';
```

we want to ignore the latter.